modules/openflashchart/graph_track_anything.php

```diff
@@ -19,8 +19,8 @@
 $fake_register_globals=false;
 //
 
-require_once(dirname(__FILE__) . "/../../interface/globals.php");
-require_once($GLOBALS['srcdir'] . "/openflashchart/php-ofc-library/open-flash-chart.php");
+require_once("../../interface/globals.php");
+require_once($modules_dir. "/openflashchart/php-ofc-library/open-flash-chart.php");
```

Comment:
$GLOBALS['modules_dir'] or the non array variable is fine.  Please just check globals.php for what is what and try to always use the lowest directory location available.
-I notice that there are a number of deprecated globals set for this in globals.php, and I will PR on a remove and cleanup of those.

Reply:
Here globals is required reason why i used `$modules_dir` either will do

Reply:
You on it.